ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/electra/helpers/MiscHelpersElectra.java

```diff
@@ -44,6 +62,20 @@ public boolean isFormerDepositMechanismDisabled(final BeaconState state) {
         .equals(BeaconStateElectra.required(state).getDepositReceiptsStartIndex());
   }
 
+  public UInt64 computeSubnetForDataColumnSidecar(UInt64 columnIndex) {
+    return columnIndex.mod(specConfigElectra.getDataColumnSidecarSubnetCount());
+  }
+
+  public Set<UInt64> computeCustodyColumnIndexes(
```

Comment:
I'm lost here.
Do you implement this one? https://github.com/ethereum/consensus-specs/blob/3dcf9ebe54cbc410fd521325bfb64ad2c474285f/specs/_features/eip7594/das-core.md?plain=1#L92-L112
I don't understand where this happens?
```python
    columns_per_subnet = NUMBER_OF_COLUMNS // DATA_COLUMN_SIDECAR_SUBNET_COUNT
```

Reply:
Nope that was actually my 'invention' solely. 
The formula for subnets is currently under active discussion o I'd leave it as is for now